public/index.html

```diff
@@ -1,29 +1,22 @@
 <!DOCTYPE html>
 <html lang="en">
-	<head>
```

Comment:
is this what the linter want or are there uncommited changes on your local branch ?

Reply:
yes, I did not do anything additionally.